docs/ide-usage.md

```diff
@@ -32,7 +32,9 @@ The following plugins are needed:
 
      ![setupguide-3.png](images/setupguide-3.png)
 
-   Note that none of the App Engine SDK or JDK to be used are required to be the `default`.
+   * Set Java SE 7 as default JDK. To do this, go to ``` Project properties → Java Compiler → Tick "Use compliance from execution environment 'JavaSE-1.7' on the 'Java Build Path'" ```.
```

Comment:
This is a wrong fix. The statement that none are required to be the `default` still holds.

What needs to be done here needs to be under "Configure the following project-specific settings (all can be found in `Project → Properties → ...`)" instead.

Also, on top of the `Java Compiler` setting, one should also ensure that the `Java Build Path` is set to JDK 7, as shown in the issue thread.